package.json

```diff
@@ -98,15 +98,15 @@
     "@craco/craco": "^6.4.5",
     "@emotion/babel-plugin": "^11.10.6",
     "@sentry/webpack-plugin": "^1.20.0",
+    "@testing-library/jest-dom": "^5.16.4",
```

Comment:
pls try to separate style / formatting / import ordering etc. changes from other changes, to make reviews of those easier

Reply:
The file was automatically formatted when upgrading dependencies. 
ok, I will restore the previous order.

Reply:
Thanks; and maybe submit a style fix pr to ensure the whole codebase is formatted according to our formatter, to avoid these in the future?

Reply:
ok